bindings/python/examples/how_tos/outputs/output_unlock_conditions.py

```diff
@@ -0,0 +1,87 @@
+from iota_sdk import *
+from dotenv import load_dotenv
+import json
+
+load_dotenv()
+
+# Create a Client instance
```

Comment:
This comment is useless if you ask me

Reply:
True that. Just copied as always :trollface: 

```suggestion
```